glue/swaybg.launcher

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 set -e
 
-background=$(snapctl get background)
+background=$(gsettings get org.gnome.desktop.background picture-uri | sed -Ee "s|[^:]*://(.*)'$|\1|")
 if [ -e "${background}" ]; then
-  export GDK_PIXBUF_MODULEDIR="$SNAP/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/gdk-pixbuf-2.0/2.10.0/loaders"
+  export GDK_PIXBUF_MODULEDIR="$SNAP/usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders"
```

Comment:
Why replace `SNAPCRAFT_ARCH_TRIPLET`? won't this fail on other architectures?

Reply:
That was an accident. Fixed